packages/terra-form-checkbox/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+* Fixed
+  * Fixed the click area is too large in Fusion theme.
```

Comment:
```suggestion
  * Fixed issue where click area of a checked checkbox is too large in Fusion theme.
```

Reply:
> 



> Can you add a gif/screen recording of the fixed behavior to the description?


https://user-images.githubusercontent.com/126649018/233263493-e752701c-5eb9-48e9-8129-ec02a4655a58.mov

i have uploaded the screen recording of the fixed behavior.

Reply:
Can you upload this video to PR description